File: cs50psets/pset6/mario/mario.py
height = input("Height: ")

while True:
    try:
        height = int(height)
        break
    except:
        height = input("Height: ")

while height < 1 or height > 8:
    height = int(input("Height: "))

for i in range(0, height):
    counter = height - i
    spaceCounter = height - 1
    while counter > 1:
        print(" ", end="")
        counter -= 1
        spaceCounter -= 1
    
    spaceCounterCopy = spaceCounter
    
    while spaceCounter >= 0:
        print("#", end="")
        spaceCounter -= 1
    
    print("  ", end="")
    
    while spaceCounterCopy >= 0:
        print("#", end="")
        spaceCounterCopy -= 1
    
    print("\n", end="")

File: cs50psets/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    int height = get_int("Height: ");
    
    while (height < 1 || height > 8)
    {
        height = get_int("Height: ");
    }

    for (int i = 0; i < height; i++)
    {
        int counter = height - i;
        int spaceCounter = height - 1;
        while (counter > 1)
        {
            printf(" ");
            counter -= 1;
            spaceCounter -= 1;
        }

        int spaceCounterCopy = spaceCounter;

        while (spaceCounter >= 0)
        {
            printf("#");
            spaceCounter -= 1;
        }

        printf("  ");

        while (spaceCounterCopy >= 0)
        {
            printf("#");
            spaceCounterCopy -= 1;
        }

        printf("\n");
    }
}